composer.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "type": "project",
   "require": {
-    "php": "~7.0.0",
```

Comment:
Let's use `~7.2.0` so that the highest available 7.2.x release is accepted (for example, I'm on 7.2.7 on my Homestead and Composer would complain with this).

Reply:
sounds good, updated!